### ISSUE TITLE
"parse error near `\n'" on terminal launch

### ISSUE THREAD

**Issue by fwesss** [withfig/fig#291]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] My issue does not relate to a specific CLI completion spec (e.g. `git checkout` is missing options in `git` completion spec). If it does, please create your issue here: [withfig/autocomplete](https://github.com/withfig/autocomplete/issues?q=)

### Summary
When launching a new terminal, the user sees:
```
Last login: Mon Jul 19 15:42:28 on ttys000
add-zle-hook-widget:187: parse error near `\n'
```

Beneath each newline is:
```
~ via 🅒 base
➜
No such widget `0:user:_zsh_highlight_widget_orig-s0.0000040000-r5143-zle-line-init'
```

### Steps to Reproduce
1. Open new shell session

### Expected Results
No parsing error and widget error

### Actual Results
Parsing and widget error

### Environment

```bash
Version 1.0.45 (B218) [U.S.]
UserShell: /usr/local/bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/alim/.fig/bin/fig
Accessibility: true
Number of specs: 142
SSH Integration: false
Tmux Integration: false
Keybindings path: /Users/alim/.fig/user/keybindings
iTerm Integration: true
Hyper Integration: false
VSCode Integration: true
Docker Integration: true
Symlinked dotfiles: true
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: /usr/local/Caskroom/miniconda/base/bin:/usr/local/Caskroom/miniconda/base/condabin:/usr/local/bin:/usr/local/miniconda3/bin:/Users/alim/bin:export:/Users/alim/.local/bin:/usr/local/opt/gettext/bin:/usr/local/opt/ncurses/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Library/Apple/usr/bin:/Users/alim/.fig/bin
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: /usr/bin/tr (4785) - ttys000
Current working directory: /Users/alim
Current window identifier: 7383/% (com.googlecode.iterm2)
PATH: /usr/local/Caskroom/miniconda/base/bin:/usr/local/Caskroom/miniconda/base/condabin:/usr/local/bin:/usr/local/miniconda3/bin:/Users/alim/bin:export:/Users/alim/.local/bin:/usr/local/opt/gettext/bin:/usr/local/opt/ncurses/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Library/Apple/usr/bin:/Users/alim/.fig/bin
FIG_INTEGRATION_VERSION: 3
```

### More Detail

shell::zsh

### Is this issue related to the docs?

- [ ] Yes
- [X] No

### Anything else?

User had 
```bash
[[ -e ~/.profile ]] && emulate sh -c 'source ~/.profile'
```
in their .zprofile.

Commenting out Fig's env variables in .profile solved his issue.

**Comment by mmartial** [comment]
Reporting a similar error for me, running `zsh` w(ith `oh-my-zsh` add `powerlines10k`)
- Also had a line with `~/.bash_profile` in my `.zprofile` 
- needed to comment the `~/.bash_profile` specific `FIG ENV VARIABLES` lines at the top and end to get it functional

**Comment by mschrage** [comment]
This is fixed as of version 1.0.47. See this [commit](https://github.com/withfig/config/commit/7be480006906b9db7c992846d4cecd90fe18f549) in withfig/config repo.
